### PR TITLE
Adds github issue template #731

### PR DESCRIPTION
**Pull request by koynro**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+Please refer to the [faqs](http://openkore.com/index.php/Frequently_Asked_Questions) (such as ["Packet Tokenizer"](http://openkore.com/index.php/Frequently_Asked_Questions#Packet_Tokenizer:_Unknown_switch:_xxxx)), [forum](http://forums.openkore.com), or [documentation wiki](http://openkore.com/index.php/Main_Page) for help using OpenKore.
+
+* Before submitting an issue, ensure you are on the latest release
```
Comment (kaliwanagan) [inline .github/ISSUE_TEMPLATE.md]: I think we should link to the doc here on how to get the latest release.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+* Check if an issue already exists for your request
+* Please post in English
```
Comment (kaliwanagan) [inline .github/ISSUE_TEMPLATE.md]: We could probably be more conversational here. Maybe something like "We know English might not be your first language; it certainly isn't for many of us too! But please do try to post in English.

**Comment by kaliwanagan** [comment]
Got a few comments, but otherwise looking good. I'm merging anyway :)

**Comment by Cozzie** [comment]
Oh nice, did not know one can do this.

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (koynro) [inline .github/ISSUE_TEMPLATE.md]: I was also thinking of making it a bit more polite 'Please post in English if possible'. We can definitely refactor it though!

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (koynro) [inline .github/ISSUE_TEMPLATE.md]: I was also considering doing this. I guess I can add this as action items to my open issue :)

**Pull request merged**